core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreUtil.java

```diff
@@ -252,4 +254,38 @@ public static void validateKeyStoreConfigName(String propertyName) throws Carbon
             throw new CarbonException("Requested key store configuration is invalid.");
         }
     }
+
+    /**
+     * This method encrypts the given password using the default crypto util.
+     *
+     * @param password Password to be encrypted.
+     * @return encrypted password.
+     */
+    public static String encryptPassword(String password) {
```

Comment:
Do we need this public static utils? Can't this be just a private method?

Reply:
Addressed with https://github.com/wso2/carbon-kernel/compare/ee7c67ca3a052b918f77f1ab62e7f98868229040..508510def3bba2d3197d9927ef37fc3d1d64c137